src/main/kotlin/com/coxautodev/graphql/tools/MethodFieldResolver.kt

```diff
@@ -149,21 +150,31 @@ open class MethodFieldResolverDataFetcher(private val sourceResolver: SourceReso
             result
         } else {
             val wrapper = options
-                .genericWrappers
```

Comment:
should cs fixes be included in feature pr?

Reply:
Im using intellij and when I push it fixes lines automatically. 
It's only cs fixes, so it doesn't matter

Reply:
Well, that actually matters because
- it makes PR more lines to read
- the PR affects lines that have nothing to do with the feature, meaning it's more difficult to find why/when the initial fix was made

Just saying, no cs configuration is included in the project anyway.

Reply:
Ok. so I won't do it next time